tools/c7n_gcp/tests/test_dns.py

```diff
@@ -41,6 +41,21 @@ def test_managed_zone_get(self):
 
         self.assertEqual(resources[0]['name'], resource_name)
 
+    def test_managed_zone_delete(self):
+        project_id = "cloud-custodian"
+        resource_name = "custodian"
+
+        factory = self.replay_flight_data('dns-managed-zone-delete', project_id)
+        p = self.load_policy(
+            {'name': 'gcp-dns-managed-zone-delete',
+             'resource': 'gcp.dns-managed-zone',
+             'filters': [{'name': resource_name}],
+             'actions': ['delete']},
+            session_factory=factory
+        )
+        resources = p.run()
+        self.assertEqual(resources[0]['name'], resource_name)
+
```

Comment:
we would generally want to refetch the resource with a client to verify the action's effect here.

Reply:
added, thanks!